engine/OthelloEngine.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, white_team_file, black_team_file, output_file, n = 8, time_li
       # turn_times keys each team's color character ('W' or 'B') to a list of their turn times
       # total time is the current sum of all of the player's turns
       self.all_moves = []
-      self.turn_number = 0
+      self.turn_number = 1
```

Comment:
Keep this as 0 please, for array indexing purposes. The first turn should be turn 0 (black's turn).

Reply:
The first turn should still be 0, as I said earlier. This is for so the array indexing works as expected, so turn 0 will be at index 0 in the array of moves

Reply:
Sorry, forgot to change it back.